README.md

```diff
@@ -498,7 +504,7 @@ tbbc_money:
 
 Update your database schema :
 ```bash
-./app/console doctrine:schema:update --force
+./bin/console doctrine:schema:update --force
```

Comment:
Maybe you should add a comment to say that the path for symfony `2.*` is `app/console`